pkg/collector/corechecks/networkpath/config.go

```diff
@@ -16,11 +16,15 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-const defaultCheckInterval time.Duration = 1 * time.Minute
+const (
+	defaultCheckInterval time.Duration = 1 * time.Minute
+	defaultTimeout       time.Duration = 10 * time.Second
```

Comment:
This essentially duplicates what we have in the traceroute runner at the moment. We should ideally have a single constant re-used everywhere

Reply:
+1